src/annotator/sidebar.coffee

```diff
@@ -72,10 +75,22 @@ module.exports = class Sidebar extends Host
       if @onLoginRequest
         @onLoginRequest()
     );
+    @crossframe.on(events.LOGOUT_REQUESTED, =>
+      if @onLogoutRequest
+        @onLogoutRequest()
+    );
     @crossframe.on(events.SIGNUP_REQUESTED, =>
       if @onSignupRequest
         @onSignupRequest()
     );
+    @crossframe.on(events.PROFILE_REQUESTED, =>
+      if @onProfileRequest
+        @onProfileRequest()
+    );
+    @crossframe.on(events.HELP_REQUESTED, =>
+      if @onHelpRequest
+        @onHelpRequest()
+    );
```

Comment:
This is getting a bit repetitive but I want to keep the code changes as dumb as possible because it's CoffeeScript